bears/rest/rstcheckBear.py

```diff
@@ -0,0 +1,55 @@
+import re
+
+from rstcheck import check
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+
+
+class rstcheckBear(LocalBear):
+    LANGUAGES = {"reStructuredText"}
+    REQUIREMENTS = {PipRequirement('rstcheck')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/ccjtkotaqzgggkkqtgelmj02x'
+
+    def run(self, filename, file, code_block_language_ignore: str,
+            ):
```

Comment:
This looks weird.
Push "code_block_language_ignore" to the next line or move ): to the above line ?